test/test-cases/custom-src-paths/__snapshots__/custom-src-paths.test.js.snap

```diff
@@ -6,13 +6,17 @@ Object {
     <meta charset=\\"UTF-8\\">
     <title>hello-world</title>
     <meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1\\">
-    
+    <link data-chunk=\\"main\\" rel=\\"preload\\" as=\\"script\\" href=\\"http://localhost:4002/runtime-27f87d9f0d19bf7deb87.js\\">
```

Comment:
This is so cool 😍

Reply:
We're losing `crossorigin="anonymous"`—is this a problem?

Reply:
☝️ Oops, commented on the wrong line—but the point still stands.

Reply:
Working on that atm. That's what the loadable-component PR is for